srcpkgs/teams/template

```diff
@@ -0,0 +1,43 @@
+# Template file for 'teams'
+pkgname=teams
+version=1.3.00.5153
+revision=1
+archs="x86_64"
+hostmakedepends="tar xz"
+short_desc="Official Microsoft Teams client"
+maintainer="Piraty <piraty1@inbox.ru>"
+license="custon:Proprietary"
+homepage="https://teams.microsoft.com/downloads"
+distfiles="https://packages.microsoft.com/repos/ms-teams/pool/main/t/teams/teams_${version}_amd64.deb"
+checksum=c928d01831b2ae9afd871172a6b56edf1d1c848655482973358f3d206375878d
+
+repository=nonfree
+#restricted=yes #really? LICENSE says it can be distributed...
+
+do_extract() {
+	ar p "${XBPS_SRCDISTDIR}/${pkgname}-${version}/teams_${version}_amd64.deb" data.tar.xz | tar --extract --xz -C .
+}
+
+do_install() {
+	# lets use a better wrapper than the provided one
+	vbin ${FILESDIR}/teams-wrapper.sh teams
+
+	vmkdir usr/lib/teams
+	vmkdir usr/share
+
+	cp -r usr/share/teams ${DESTDIR}/usr/lib/
+
+	for item in applications pixmaps ; do
+		cp -r usr/share/$item ${DESTDIR}/usr/share/$item
+	done
```

Comment:
If its just two items wouldn't it be better to remove the loop and replace it with single statements?

Reply:
Also why not use `vcopy` instead of `cp -r`.